ts/Series/Sankey/SankeySeriesDefaults.ts

```diff
@@ -243,6 +243,17 @@ const SankeySeriesDefaults: PlotOptionsOf<SankeySeries> = {
      */
     minLinkWidth: 0,
 
+    /**
+     * Determines which side of the chart the nodes are to be aligned to.
+     *
```

Comment:
We also need to explain that in an inverted chart, `top` transforms to `left`, and `bottom` transforms to `right`. Or is there some other wording we could use that worked either way? Like `start`, `center`, `end` or something, except start and end don't really make sense either...